docs/pages/appbars/index.vue

```diff
@@ -0,0 +1,188 @@
+      </DocsShow>
+
+      <h3>Bottom bars</h3>
+      <p>Bottom bars are placed on pages that house editing, content selection, or multi-step processes. They may contain buttons that allow the user to submit specific action(s) or continue with the process. Bottom bars can also contain contextual information and error messaging relevant to the task</p>
+      <DocsShow>
+        <img class="modal-img" src="./bottom-1.png">
+      </DocsShow>
+    </DocsPageSection>
+
+    <DocsPageSection title="Layout" anchor="#layout">
+      <ul>
+        <li>Place navigation icons on the far left</li>
+        <li>Place titles to the right of the navigation icon</li>
+        <li>Place contextual icons to the far right</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Titles" anchor="#titles">
+      <p>Appbars must have titles that describe high-level information.</p>
+      <ul>
+        <li>Titles can describe a feature a user is engaged in (e.g. Create Quiz)</li>
+        <li>Titles can describe a high-level section of an app (e.g. Learn, Coach)</li>
+        <li>Titles can describe an application being used (e.g. Kolibri Studio)</li>
+        <li>Titles should never include names of user generated text (e.g. resource title or username)</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsDoNot>
+      <template v-slot:do>
+        <img src="./do-1.png">
+        <p class="do-dont">
+          Use general titles in app bars
+        </p>
+      </template>
+      <template v-slot:not>
+        <img src="./dont-1.png">
+        <p class="do-dont">
+          Use user-generated text in app bars
+        </p>
+      </template>
+    </DocsDoNot>
+
+    <DocsPageSection title="Exceptions" anchor="#exceptions">
+      <p>Purposeful exceptions can be made to incorporate special elements into top bars</p>
+    </DocsPageSection>
+
+    <DocsDoNot style="display: inline-block;">
+      <template v-slot:do>
+        <img src="./exception-1.png">
+        <p class="do-dont">
+          Appbar color may change to white for critical workspaces
+        </p>
+      </template>
+    </DocsDoNot>
+
+    <DocsDoNot style="display: inline-block;">
+      <template v-slot:do>
+        <img src="./exception-2.png">
+        <p class="do-dont">
+          Learning aids and indicators can be placed in app bars
+        </p>
+      </template>
+    </DocsDoNot>
+
+    <DocsDoNot style="display: inline-block;">
+      <template v-slot:do>
+        <img src="./exception-3.png">
+        <p class="do-dont">
+          Search text fields can be placed in app bars when global searching is a critical feature
+        </p>
+      </template>
+    </DocsDoNot>
+
+    <DocsDoNot style="display: inline-block;">
+      <template v-slot:do>
+        <img src="./exception-4.png">
+        <p class="do-dont">
+          For resource pages in the Learn app, use a back link at the top of the page instead of an app bar
```

Comment:
Is this tracked as an issue? I know this is what we want to move towards, but we should make sure it describes the current, or soon to be, state of things